lib/logger/lib/logger.ex

```diff
@@ -178,37 +178,17 @@ defmodule Logger do
   Erlang's logging functionalities. All the configurations below must
   be set before the `:logger` application starts.
 
-    * `:handle_otp_reports` - redirects OTP reports to `Logger` so
-      they are formatted in Elixir terms. This effectively disables
-      Erlang standard logger. Defaults to `true`.
```

Comment:
This option has not been actually removed. We still need to know if we should remove Erlang handlers or not. So please add this line and the next removed paragraph back.